src/test/java/marquez/client/MarquezHttpTest.java

```diff
@@ -223,4 +227,14 @@ public void testGet_throwsOnHttpError() throws Exception {
     final URL url = marquezUrl.from(path("/namespace/%s", newNamespaceName()));
     assertThatExceptionOfType(MarquezHttpException.class).isThrownBy(() -> marquezHttp.get(url));
   }
+
+  @Test
+  public void testClose() throws Exception {
+    // the sclient is not closeable
+    marquezHttp.close();
```

Comment:
_Minor:_ Does this line affect the test, or is it more for documentation?

Reply:
it does cover that close() is  a no op when the client is not closeable. Notably it doesn't throw an exception.

Reply:
I see. We could move the assertion into its own test if we wanted to make it more explicit, but either way works 👍